script.js

```diff
@@ -0,0 +1,25 @@
+var block = document.getElementById("test");
+var left = 0;
+var top1 = 0;
+var bottom = 0;
+var right = 0;
+
+document.onkeydown = function (event){
```

Comment:
instead of onkeydown add
`document.addEventListerer('keydown', funtion() => {...})`

Reply:
use arrow functions 
`(event ) => {}` instead of `function(event){}`